modules/kubernetes-addons/crossplane/main.tf

```diff
@@ -14,5 +14,52 @@ module "helm_addon" {
   helm_config       = local.helm_config
   irsa_config       = null
 
-  depends_on = [kubernetes_namespace_v1.irsa]
+  depends_on = [kubernetes_namespace_v1.crossplane]
+}
+
+resource "kubectl_manifest" "controller_config" {
```

Comment:
Same concern that I brought up in the Karpenter example PR. Should we consider using tfk8s which is terraform recommended and use the Terraform kubernetes provider?

Reply:
done

Reply:
See my response on Karpenter PR. We are using only `kubectl_manifest` in our solution